recipes/iguana/all/conanfile.py

```diff
@@ -0,0 +1,78 @@
+from conan import ConanFile
+from conan.errors import ConanInvalidConfiguration
+from conan.tools.build import check_min_cppstd
+from conan.tools.files import copy, get
+from conan.tools.layout import basic_layout
+from conan.tools.scm import Version
+import os
+
+required_conan_version = ">=1.52.0"
+
+class IguanaConan(ConanFile):
+    name = "iguana"
+    description = "universal serialization engine"
+    license = "Apache-2.0"
+    url = "https://github.com/conan-io/conan-center-index"
+    homepage = "https://github.com/qicosmos/iguana"
+    topics = ("serialization", "json", "xml", "header-only")
+    package_type = "header-library"
+    settings = "os", "arch", "compiler", "build_type"
+    no_copy_source = True
+
+    @property
+    def _min_cppstd(self):
+        return 20
+
+    @property
+    def _compilers_minimum_version(self):
+        return {
+            "gcc": "11",
+            "clang": "12",
+            "apple-clang": "14",
+            "Visual Studio": "16",
+            "msvc": "192",
+        }
+
+    def layout(self):
+        basic_layout(self, src_folder="src")
+
+    def requirements(self):
+        self.requires("frozen/1.1.1", transitive_headers=True)
```

Comment:
Note that for `package_type = "header-library"` recipes, `transitive_headers` and `transitive_libs` are both True by default